app/src/main/java/org/mozilla/fenix/perf/Performance.kt

```diff
@@ -37,4 +44,22 @@ object Performance {
             activity.reportFullyDrawn()
         }
     }
+
+    fun processIntentIfPerformanceTest(intent: Intent, context: Context) {
+
+        if (intent.getBooleanExtra(HomeActivity.EXTRA_IS_PERFORMANCE_TEST, false)) {
+            val batteryStatus = context.registerReceiver(null, IntentFilter(Intent.ACTION_BATTERY_CHANGED))
+            val isPhonePlugged = batteryStatus!!.getIntExtra(BatteryManager.EXTRA_PLUGGED, -1)
+            val isAdbenabled = android.provider.Settings.Global.getInt(context.getContentResolver(),
+                                                android.provider.Settings.Global.ADB_ENABLED, 0)
+
+            if ((isPhonePlugged == BatteryManager.BATTERY_PLUGGED_USB) && (isAdbenabled == 1)) {
+                FenixOnboarding(context).finish()
+
+                for (i in 0..Settings.trackingProtectionOnboardingMaximumCount) {
```

Comment:
Rather than incrementing here, can we just add an override directly into Settings? It'd reduce the impact on startup time and be clearer.